openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -78,64 +79,59 @@ def _get_response(query_param, endpoint=ENDPOINT, retries=RETRIES):
         return response_json, total_images
 
 
+def get_int_value(data, key):
```

Comment:
Do you mind adding type annotations to these functions? 😄 Just so we know what types should be considered valid